.changeset/moody-rocks-accept.md

```diff
@@ -0,0 +1,28 @@
+---
+'@backstage/plugin-catalog': patch
+'@backstage/plugin-catalog-react': patch
```

Comment:
I'd say this is very much a breaking change as is right? Perhaps we should make opt-in for now?